### PR TITLE
Update Firebolt features and connection parameters

### PR DESCRIPTION
**Pull request by ptiurin**
## What are you changing in this pull request and why?

Firebolt has rolled out a new authentication method with service account credentials, all new Firebolt DBT users would use it from now on. Existing dbt-firebolt connector handles this behind the scenes allowing users that haven't migrated yet to use their old credentials while also allowing new customers to use the recommended way of authentication. This change in documentation reflects the new auth to avoid confusion for the new users.

Also updating some of the outdated information on Firebolt features supported, like removal of Join indexes and engine types and updating doc links to the new website.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

Adding or removing pages (delete if not applicable):
- [x] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/reference/resource-configs/firebolt-configs.md [hunk 1]
```diff
@@ -180,54 +172,27 @@ models:
...
-| `table_type`       | Whether the materialized table will be a [fact or dimension](https://docs.firebolt.io/working-with-tables.html#fact-and-dimension-tables) table. |
-| `indexes`          | A list of join indexes to create on the dimension table. |
-| `type`             | Specifies whether the index is an aggregating index or join index. Aggregating indexes only apply to fact tables, so for dimension tables set to `join`. |
-| `join_column`      | Sets the join key of the join index using the inputted column name from the model. |
-| `dimension_column` | Sets the columns to be loaded into memory on the join index using the inputted list of column names from the mode. |
+| `table_type`       | Whether the materialized table will be a [fact or dimension](https://docs.firebolt.io/godocs/Overview/working-with-tables/working-with-tables.html#fact-and-dimension-tables) table. |
 
 
-#### Example of a Dimension Table With a Join Index
+## How Aggregating Indexes Are Named
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/firebolt-configs.md:193]:
```suggestion
## How aggregating indexes are named
```

Comment (mirnawong1) [inline website/docs/reference/resource-configs/firebolt-configs.md:193]:
```suggestion
## How aggregating indexes are named
```

Comment (mirnawong1) [inline website/docs/reference/resource-configs/firebolt-configs.md:193]:
```suggestion
## How aggregating indexes are named
```

Comment (mirnawong1) [inline website/docs/reference/resource-configs/firebolt-configs.md:193]:
```suggestion
## How aggregating indexes are named
```

**Review by mirnawong1 — APPROVED** [review]
thanks for opening this up @ptiurin , will get this merged for you!

**Comment by mirnawong1** [comment]
hey @ptiurin , thanks for opening this pr! I'm trying to update this branch but i don't have permissions. any chance you can update the branch on your end before i merge?

**Comment by ptiurin** [comment]
Hi @mirnawong1 , sorry I was OOO so didn't catch your comment in time. Thanks for merging this PR! Do you want me to create another with the changes proposed here?

**Pull request merged**